### PR TITLE
Temporarily support both JAVA_IMPL and JDK_IMPL

### PR DESCRIPTION
**Pull request by renfeiw**
- Convert JAVA_IMPL to JDK_IMPL if JAVA_IMPL is defined
- Change JAVA_IMPL in playlist.xml to JDK_IMPL

[ci skip]

Signed-off-by: Renfei Wang <renfeiw@ca.ibm.com>

### CODE REVIEW

**Comment by renfeiw** [comment]
reviewer: @llxia

File: test/TestConfig/settings.mk [hunk 1]
```diff
@@ -257,6 +265,7 @@ setup_%:
 		$(ECHO) There are $(TOTALCOUNT) test targets in $(TESTTARGET).; \
 	fi
 	$(JAVA_COMMAND) -version
+	$(CMD)
```
Comment (llxia) [inline test/TestConfig/settings.mk]: Is this intended?

Comment (renfeiw) [reply test/TestConfig/settings.mk]: removed

**Comment by llxia** [comment]
Jenkins test sanity plinux all

**Comment by llxia** [comment]
fyi @AdamBrousseau @jdekonin @vsebe

**Comment by AdamBrousseau** [comment]
I'm guessing we will need to check all the test jobs are using `JDK_IMPL` and `JDK_VERSION` as default parameters?

**Comment by llxia** [comment]
yes, just a heads up for now, Jenkins file in AdoptOpenJDK needs to be updated to use JDK_IMPL and JDK_VERSION. Maybe support both for the time being.

**Pull request merged**